lib/manageiq/api/common/open_api/generator.rb

```diff
@@ -176,6 +204,14 @@ def openapi_list_description(klass_name, primary_collection)
                       "schema" => { "$ref" => build_collection_schema(klass_name) }
                     }
                   }
+                },
+                "404" => {
```

Comment:
I think we want to skip this if `primary_collection` is `nil`, right?  We should only have a 404 if there is an id in the path.